### ISSUE TITLE
Val loss vray greatly during training

### ISSUE THREAD

**Issue by jiusansan222** [nii-yamagishilab/mos-finetune-ssl#3]
When I used this code and bvcc dataset to trian a mos predict model, I found that the results of serval times were different. And I plot the train loss and val loss picture, it shows that val loss vray greatly from the vary begining of training. Do you have the same findings? Is it overfitting? Thanks!!

![image](https://user-images.githubusercontent.com/58416364/193957740-998792d8-3cd6-4de9-90e0-d951c387d546.png)
![image](https://user-images.githubusercontent.com/58416364/193957769-e06dbbfd-7966-4eb6-b9d3-7bcd88e4534a.png)

**Comment by seblemaguer** [comment]
The validation loader is shuffled.

You can replace this:
```python
validloader = DataLoader(validset, batch_size=2, shuffle=True, num_workers=2, collate_fn=validset.collate_fn)
```

by this:
```python
validloader = DataLoader(validset, batch_size=2, shuffle=False, num_workers=2, collate_fn=validset.collate_fn)
```

and see how it goes
